a.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
```

Comment:
Pretty sure this shouldn't be here? Would recommend erasing it from git history.

Reply:
oops, my bad